src/patternfly/patternfly-date-picker.scss

```diff
@@ -0,0 +1,177 @@
+    --pf-c-date-picker__flatpickr__month-select--FontWeight: var(--pf-global--FontWeight--normal);
+    --pf-c-date-picker__flatpickr__month-select--BackgroundUrl: url("data:image/svg+xml;charset=utf8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 320 512'%3E%3Cpath fill='%23urrentColor' d='M31.3 192h257.3c17.8 0 26.7 21.5 14.1 34.1L174.1 354.8c-7.8 7.8-20.5 7.8-28.3 0L17.2 226.1C4.6 213.5 13.5 192 31.3 192z'/%3E%3C/svg%3E");;
+    --pf-c-date-picker__flatpickr__month-select--BackgroundSize: .625em;
+    --pf-c-date-picker__flatpickr__month-select--BackgroundPosition: calc(100% - var(--pf-global--spacer--sm)) center;
+    --pf-c-date-picker__flatpickr__month-select--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
+    --pf-c-date-picker__flatpickr__month-select--hover--BackgroundColor: var(--pf-c-date-picker__flatpickr__month-select--BackgroundColor);
+    --pf-c-date-picker__flatpickr__month-select--year--MarginLeft: var(--pf-global--spacer--sm);
+    --pf-c-date-picker__flatpickr__month-arrow--Width: 7px;
+    --pf-c-date-picker__flatpickr__month-arrow--Color: var(--pf-global--Color--dark-200);
+    --pf-c-date-picker__flatpickr__month-arrow--hover--Color: var(--pf-global--Color--dark-100);
+    --pf-c-date-picker__flatpickr__year--hover--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
+    --pf-c-date-picker__flatpickr__year--FontWeight: var(--pf-global--FontWeight--normal);
+
+    // Weekdays section
+    --pf-c-date-picker__flatpickr__weekdays--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
+    --pf-c-date-picker__flatpickr__weekdays--PaddingBottom: var(--pf-global--spacer--md);
+    --pf-c-date-picker__flatpickr__weekdays--MarginBottom: var(--pf-global--spacer--sm);
+    --pf-c-date-picker__flatpickr__weekdays--BorderBottom: var(--pf-global--BorderWidth--sm) solid var(--pf-global--BorderColor--100);
+    --pf-c-date-picker__flatpickr__weekday--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
+
+    // Calendar days
+    --pf-c-date-picker__flatpickr__calendar__day--Color: var(--pf-global--Color--dark-100);
+    --pf-c-date-picker__flatpickr__calendar__day--disabled--Color: var(--pf-global--disabled-color--200);
+    --pf-c-date-picker__flatpickr__calendar__day--hover--BorderColor: var(--pf-global--BackgroundColor--light-300);
+    --pf-c-date-picker__flatpickr__calendar__day--hover--BackgroundColor: var(--pf-global--BackgroundColor--light-300);
+    --pf-c-date-picker__flatpickr__calendar__day--selected--hover--BorderColor: var(--pf-global--primary-color--100);
+    --pf-c-date-picker__flatpickr__calendar__day--selected--hover--Background: var(--pf-global--primary-color--100);
+    --pf-c-date-picker__flatpickr__calendar__day--selected--hover--BorderColor: var(--pf-global--Color--light-100);
+    --pf-c-date-picker__flatpickr__calendar__day--today--BorderColor: var(--pf-global--primary-color--100);
```

Comment:
today was a little confusing what about currentday

Reply:
We can but the flatpickr library uses `.flatpickr-day.today`, so I'm just re-using their names. They may start using "current day" to represent something else along with "today", then we might end up with conflicts in the naming. Seems best to match what they have if we can.